.github/workflows/deploy-staging.yml

```diff
@@ -38,10 +39,12 @@ jobs:
         with:
           filters: |
             api:
+              - "packages/shared/
               - "packages/api/**"
               - "packages/api-sdk/**"
               - "packages/commonwell-sdk/**"
               - "packages/core/**"
+              - "packages/ihe-gateway-sdk/**"
```

Comment:
Ditto Changes on this package don't need to result in a new deploy of the API, right?